src/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -94,11 +95,13 @@ export default function RecentTransactionList({ accountsList, className = '' }:
         .map(({ txType, ...tx }) => ({ ...tx, txType: txType as TxType })),
     [txHistory, accountsList.length]
   );
-
+  const accountsMap = accountsList.reduce((acc, cur) => {
```

Comment:
Why are we not using a list here?

Reply:
Using a map instead of searching an array changes search speed for an individual item to `o(1)` from `o(n)` so overall speed goes from roughly `o(m*n)` to `o(m+n)`.